.github/labeler.yml

```
@@ -3,24 +3,30 @@
 "Type: Bug":
   - head-branch: [ '^bug', '-bug-' ]
 "Type: Maintenance":
-  - head-branch: [ '^maint', '-maintenance-' ]
+  - head-branch: [ '^maint', '^refact', '^chore', '-maintenance-' ]
```

YAMLlint (1.37.1)

[error] too many spaces inside brackets

(brackets)